src/App.js

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import "./styles.css";
+import ConvertToDate from "./Util.js";
```

Comment:
in react, we usually us `Captital` as component, and others use lower case.
so recommend use `convertToDate` and `util.js`

Reply:
if you take `ConvertToDate` as a react component, you could ignore this.

Reply:
ok, I will modify it to pure JS function.